Doc/library/array.rst

```diff
@@ -79,14 +79,16 @@ The module defines the following type:
 .. class:: array(typecode[, initializer])
 
    A new array whose items are restricted by *typecode*, and initialized
-   from the optional *initializer* value, which must be a list, a
-   :term:`bytes-like object`, or iterable over elements of the
-   appropriate type.
+   from the optional *initializer* value, which must be a :class:`bytes`
+   or :class:`bytearray` object, a unicode string, or iterable over elements
+   of the appropriate type.
```

Comment:
Only bytes and bytearray are special, bytes-like objects in general are not supported. List is a particular type of iterable.